Assets/MRTK/SDK/Experimental/NonNativeKeyboard/Scripts/NonNativeKeyboard.cs

```diff
@@ -172,6 +172,16 @@ public enum LayoutType
         /// </summary>
         public event Action<bool> OnKeyboardShifted = delegate { };
 
+        /// <summary>
+        /// Event fired when char key on keyboard is pressed.
+        /// </summary>
+        public event Action<KeyboardValueKey> OnKeyboardValueKeyPressed = delegate { };
+
+        /// <summary>
+        /// Event fired when function key on keyboard is pressed.
+        /// </summary>
+        public event Action<KeyboardKeyFunc> OnKeyboardKeyFuncPressed = delegate { };
```

Comment:
OnKeyboardFunctionKeyPressed?

I think the delta in characters isn't that much (i.e. four letters)

Also reversing the order of Key/Func to mirror ValueKey pressed